### PR TITLE
Add carbon-utils feature

### PR DESCRIPTION
**Pull request by thusithathilina**

### CODE REVIEW

**Comment by thusithathilina** [comment]
Resolves https://github.com/wso2/carbon-utils/issues/5

File: features/org.wso2.carbon.utils.feature/pom.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                                </bundle>
+                            </bundles>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (Kishanthan) [inline features/org.wso2.carbon.utils.feature/pom.xml]: EOF line is missing

**Review by Kishanthan — CHANGES_REQUESTED** [review]

File: features/org.wso2.carbon.utils.feature/pom.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.carbon.utils.feature</artifactId>
+    <packaging>carbon-feature</packaging>
+    <name>WSO2 Carbon Utils - Carbon Utils Feature</name>
+    <url>http://wso2.com</url>
+    <description>This feature contains the carbon-utils</description>
+
+
+    <dependencies>
```
Comment (vidurananayakkara) [inline features/org.wso2.carbon.utils.feature/pom.xml]: Should we remove the blank lines here?

File: features/org.wso2.carbon.utils.feature/pom.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.carbon.utils.feature</artifactId>
+    <packaging>carbon-feature</packaging>
+    <name>WSO2 Carbon Utils - Carbon Utils Feature</name>
+    <url>http://wso2.com</url>
+    <description>This feature contains the carbon-utils</description>
+
+
```
Comment (vidurananayakkara) [inline features/org.wso2.carbon.utils.feature/pom.xml]: Should we remove the blank lines here?

File: features/org.wso2.carbon.utils.feature/pom.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.utils</groupId>
+            <artifactId>carbon-utils</artifactId>
+        </dependency>
+    </dependencies>
+
```
Comment (vidurananayakkara) [inline features/org.wso2.carbon.utils.feature/pom.xml:40]: Same as the above comment. Should we remove the blank lines here?

**Review by Kishanthan — APPROVED** [review]

File: features/org.wso2.carbon.utils.feature/pom.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        <artifactId>carbon-utils-parent</artifactId>
+        <groupId>org.wso2.carbon.utils</groupId>
+        <version>1.2.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.carbon.utils.feature</artifactId>
+    <packaging>carbon-feature</packaging>
+    <name>WSO2 Carbon Utils - Carbon Utils Feature</name>
```
Comment (Kishanthan) [inline features/org.wso2.carbon.utils.feature/pom.xml]: This name is not consistent with other maven modules in this. See below build summary.

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] WSO2 Carbon - Utils Parent ......................... SUCCESS [  4.901 s]
[INFO] WSO2 Carbon - Utils ................................ SUCCESS [ 11.640 s]
[INFO] WSO2 Carbon Utils - Carbon Utils Feature ........... SUCCESS [  2.032 s]
[INFO] ------------------------------------------------------------------------

**Review by Kishanthan — CHANGES_REQUESTED** [review]

**Review by Kishanthan — APPROVED** [review]

**Pull request merged**